dashboards-observability/public/components/event_analytics/explorer/explorer.tsx

```diff
@@ -993,8 +1016,8 @@ export const Explorer = ({
       setToast('Name field cannot be empty.', 'danger');
       return;
     }
-    if (!isValidDataConfigOptionSelected) {
-      setToast('Invalid value options configuration selected.', 'danger');
+    if (!Boolean(isValidValueOptionConfigSelected)) {
```

Comment:
do we need to convert to boolean here? any bug this change is trying to address?

Reply:
There was no need of Boolean, removed it.